scss-partials/_service.scss

```diff
@@ -0,0 +1,47 @@
+.service {
+    align-items: center;
+    text-align: center;
+    margin: 100px 0px 48px;
+
+    @include media ( "<=sm" ) {
+        margin: 64px 0px 0px;
+    }
+    
+    &__header {
+        color: $tws_black;
+        font-family: $font-heading;
+        font-style: normal;
+        font-size: 24px;
+        font-weight: 300;
+        line-height: 32px;
+        margin: 0px 16px;
+
+        @include media ( ">=md" ) {
+            font-size: 32px;
+            line-height: 40px;
+        }
+    }
+
+    &__info {
+        
+        &-text {
+            color: $tws_black;
+            font-family: $font-regular;
+            font-size: 16px;
+            line-height: 24px;
+            font-weight: 400;
+            max-width: 343px;
```

Comment:
why would you like to limit it so much? If you based it on mobile design, it's probably because that's the text width that fit into a small screen presented there but it doesn't mean it should have such a restricted width :) I'd say make it around max-width: 600px for all screen widths and add horizontal padding (16px probably). The text will nicely adapt to any device width then and will take up max 600px on desktop

Reply:
Yeah I think I've done this from looking at the mobile design on figma and not adjusting for desktop etc. Will change this!

Reply:
Oh yeah I've just seen below I added in the max-width to be 600px when the screen size is larger than small screen...